### DISCUSSION TITLE
my macOS is 14.4 ,when I use brew ,show error : "Version value must be a string; got a NilClass ()"

### DISCUSSION THREAD

**Discussion by pass13046132** [Homebrew/discussions#5348] [Everyday usage]
### Output of `brew config`

```shell
1
```

### Output of `brew doctor`

```shell
/usr/local/Homebrew/Library/Homebrew/version.rb:369:in `initialize': Version value must be a string; got a NilClass () (TypeError)
```

### Description of issue

how to fix it.? 

thanks

**Comment by SMillerDev** [comment]
If `brew config` only outputs `1`, something is really broken in your Homebrew install. I would recommend running `brew update-reset` and if that doesn't work I'd recommend reinstalling Homebrew.
